docs/add-a-sut.md

```diff
@@ -0,0 +1,113 @@
+# Tutorial: Adding A New SUT (System Under Test)
+
+Adding a new SUT to ModelBench can be done in a number of ways, but here is an example of the easiest. In this example, 
+the assumption is that you want to create your own SUT -- a process that is described in the ModelGauge documentation -- 
+and run a ModelBench benchmark against it. Here are the steps to do so.
+
+## 1. Install ModelBench
+
+First things first, install ModelBench and make sure it works. For installation instructions, look [here](https://github.com/mlcommons/modelbench?tab=readme-ov-file#modelbench) 
+and [here, specifically](https://github.com/mlcommons/modelbench?tab=readme-ov-file#trying-it-out) for trying it out 
+instructions.
+
+This tutorial will assume you've installed ModelBench using PyPi
+
+## 2. Create your SUT plugin
+
+For the purposes of this tutorial, please refer to the **[Creating a basic SUT](https://github.com/mlcommons/modelgauge/blob/main/docs/tutorial_suts.md#creating-a-basic-sut)** 
+tutorial in the ModelGauge repository. Ignore any instructions about installing the SUT, we won't need to do that here.
+
+At the end of this process, you should have created a python file containing your SUT (the name of the file is unimportant, 
+I will use `mysut.py` for this tutorial example) containing a line that will register the SUT with ModelGauge like the 
+following:
+
+```python
+SUTS.register(DemoYesNoSUT, "demo_yes_no")
+```
+
+If you don't want to follow the instructions on creating a basic SUT right now, don't worry about it! Use this:
+
+```python
+from modelgauge.prompt import ChatPrompt, TextPrompt
+from modelgauge.prompt_formatting import format_chat
+from modelgauge.sut import PromptResponseSUT, SUTCompletion, SUTResponse
+from modelgauge.sut_capabilities import AcceptsChatPrompt, AcceptsTextPrompt
+from modelgauge.sut_decorator import modelgauge_sut
+from modelgauge.sut_registry import SUTS
+from pydantic import BaseModel
+
+class DemoYesNoRequest(BaseModel):
+    text: str
+
+class DemoYesNoResponse(BaseModel):
+    number_of_words: int
+    text: str
+
+@modelgauge_sut(capabilities=[AcceptsTextPrompt, AcceptsChatPrompt])
```

Comment:
This code is the culmination of all of the code snippets from the ModelBench [Creating a basic SUT](https://github.com/mlcommons/modelgauge/blob/main/docs/tutorial_suts.md#creating-a-basic-sut) example here (+ some necessary imports that are missing from that example). I'd be happy to simplify this example if we also simplify that one in the same way.

What I'm trying to avoid is users looking at that example, and looking at this example, and trying to figure out why they're different.